pkgs/joblib.yaml

```diff
@@ -0,0 +1,7 @@
+# Joblib version 0.9.0b2 for simple prallel python loops.
+
+extends: [setuptools_package]
+
+sources:
+  - key: tar.gz:ggsl323ify27x6na4zsbbo76ealp7vis
+    url: https://pypi.python.org/packages/source/j/joblib/joblib-0.9.0b2.tar.gz#md5=530ead86cc3d07aa42d0db65adbc4dd8
```

Comment:
The `#md5=530ead86cc3d07aa42d0db65adbc4dd8` part of the `url` is not needed, but otherwise it looks good.